pennylane/drawer/tape_mpl.py

```diff
@@ -189,36 +189,41 @@ def tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, **kwarg
 
     **Formatting:**
 
-    You can globally control the style with ``plt.rcParams`` and styles, see the
-    `matplotlib docs <https://matplotlib.org/stable/tutorials/introductory/customizing.html>`_ .
-    If we customize ``plt.rcParams``, we get a
-    different style:
+    PennyLane has inbuilt styles. All available styles can be determined by
+    evaluating ``qml.drawer.available_styles()``. Any available string
+    can then be passed to ``qml.drawer.use_style``.
 
     .. code-block:: python
 
-        plt.rcParams['patch.facecolor'] = 'white'
-        plt.rcParams['patch.edgecolor'] = 'black'
-        plt.rcParams['patch.linewidth'] = 2
-        plt.rcParams['patch.force_edgecolor'] = True
-        plt.rcParams['lines.color'] = 'black'
+        qml.drawer.use_style('black_white')
 
         fig, ax = tape_mpl(tape)
```

Comment:
Remove empty line as it shows up in the rendered docs

```suggestion
        fig, ax = tape_mpl(tape)
```

![image](https://user-images.githubusercontent.com/21964864/144497543-44f772fb-29f3-4f01-9af0-92baa34ff31d.png)

Reply:
this style looks so good 😍